src/components/textarea.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(app) {
             }
             return 'formio/components/texteditor.html';
           }
+          else if (!$scope.readOnly && $scope.component.components) {
```

Comment:
Yeah, I hate to be a stickler, but I think I would like to request that they are called something else besides ```components```. The reason is because right now we already have a property of ```components``` that can be attached to any component that means something else. That currently means, these are components within this component (like for Panels, Tables, Columns, etc). I would be willing to accept this if we change it to say ```editorComponents``` since that seems to be what they are used for.